spec/features/admin/views/admin_can_see_all_orders_spec.rb

```diff
@@ -7,16 +7,23 @@
     allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
   end
 
+
+
   context "As an admin and two orders in the database" do
     let!(:order_1) { create(:order, status: "ordered") }
     let!(:order_2) { create(:order) }
 
-    it "I can see the total number of orders for each status" do
+    xit "I can see the total number of orders for each status" do
+    end
+
+    it "I can see all orders with associated attributes" do
       visit admin_dashboard_index_path
 
       expect(page).to have_content(order_1.id)
       expect(page).to have_content(order_1.date)
       expect(page).to have_content(order_1.status.capitalize)
+      expect(page).to have_content(order_1.total_price)
+      expect(page).to have_content(order_2.total_price)
```

Comment:
@GabrielAfflitto How do we know if `total_price` is calculating the correct amount? Like, what happens if it returns a string of `"I only drink water out of champaign flutes."` This test will still pass but that's not really what we want. Not sure this is testing the actual business value. // @Maxscores